webapp/src/main/java/com/box/l10n/mojito/service/tm/TMService.java

```diff
@@ -648,9 +651,21 @@ && isUpdateNeededForTmTextUnitVariant(
           currentTmTextUnitVariant.setStatus(status);
           currentTmTextUnitVariant.setIncludedInLocalizedFile(includedInLocalizedFile);
           tmTextUnitVariantRepository.save(currentTmTextUnitVariant);
+          tmTextUnitVariantCommentService.addComment(
+              currentTmTextUnitVariant,
+              TMTextUnitVariantComment.Type.AI_TRANSLATION,
+              TMTextUnitVariantComment.Severity.INFO,
+              "AI translation review completed with no updates.");
```

Comment:
Nit: Can we change this text to `AI translation review was accepted`?
`review completed with no updates` conveys the impression that this text unit was reviewed but not updated at all, where in reality its status was updated (and they were happy with it)
Very pedantic, but I can see the confusion this could cause